src/me/fru1t/subedit/utils/Utils.kt

```diff
@@ -45,4 +45,13 @@ object Utils {
   }
 
   fun askYes(): Boolean = readLine()?.toLowerCase() == "y"
+  fun askForFontSize(): Int {
```

Comment:
Maybe `askForInt()` to make it a bit more general and either move the println to the caller OR pass in the prompt as a parameter